proto/cosmos/upgrade/v1beta1/upgrade.proto

```diff
@@ -43,6 +43,10 @@ message Plan {
   google.protobuf.Any upgraded_client_state = 5 [deprecated = true];
 }
 
+// TODO HV2: this was deleted from heimdall's gov/types/proposal.go.
```

Comment:
nit: should this be a TODO comment ?

Reply:
Right, pushed with commit 1d959ccd8417b6a339276acc5e444be3de54591a. thanks